config/environments/production.rb

```diff
@@ -64,39 +69,17 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Log disallowed deprecations.
-  config.active_support.disallowed_deprecation = :log
-
-  # Tell Active Support which deprecation messages to disallow.
-  config.active_support.disallowed_deprecation_warnings = []
-
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require "syslog/logger"
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
-
-  if ENV.fetch('RAILS_LOG_TO_STDOUT', nil).present?
-    logger           = ActiveSupport::Logger.new($stdout)
-    logger.formatter = config.log_formatter
-    config.logger    = ActiveSupport::TaggedLogging.new(logger)
-  end
-
-  # see https://github.com/shadabahmed/logstasher
-  config.logstasher.enabled = true
```

Comment:
Are we deliberately switching off logstasher in production?

Reply:
no,
I think rails update removed it, and I forgot to put it back

Reply:
done!